src/biogeochem/CNPhenologyMod.F90

```diff
@@ -1841,34 +1908,44 @@ subroutine CropPhenology(num_pcropp, filter_pcropp                     , &
             !         According to Chris Kucharik, the dataset of
             !         xinpdate was generated from a previous model run at 0.5 deg resolution
 
+
+            ! Only allow sowing according to normal "window" rules if not using prescribed
+            ! sowing dates at all, or if this cell had no values in the prescribed sowing
+            ! date file.
+            allow_unprescribed_planting = (.not. use_cropcal_streams) .or. crop_inst%rx_sdates_thisyr(p,1)<0
+
             ! winter temperate cereal : use gdd0 as a limit to plant winter cereal
 
             if (ivt(p) == nwwheat .or. ivt(p) == nirrig_wwheat) then
 
                ! Are all the normal requirements for planting met?
-               do_plant_normal = a5tmin(p)   /= spval                  .and. &
+               do_plant_normal = allow_unprescribed_planting           .and. &
+                                 a5tmin(p)   /= spval                  .and. &
                                  a5tmin(p)   <= minplanttemp(ivt(p))   .and. &
                                  jday        >= minplantjday(ivt(p),h) .and. &
+                                 jday        <= maxplantjday(ivt(p),h) .and. &
                                  (gdd020(p)  /= spval                  .and. &
                                  gdd020(p)   >= gddmin(ivt(p)))
                ! If not, but it's the last day of the planting window, what about relaxed rules?
-               do_plant_lastchance = (.not. do_plant_normal)               .and. &
-                                     jday       >=  maxplantjday(ivt(p),h) .and. &
```

Comment:
Fixing this behavior. Previously, the relaxed planting criteria were allowed on *or after* the last day of the planting window (within the same calendar year). The fixed version allows relaxed criteria only on the last day.